modules/mobile/spec/request/user_request_spec.rb

```diff
@@ -290,6 +293,73 @@
           )
         end
       end
+
+      context 'with a user who does not have access to schedule appointments' do
+        context 'due to not having any registered faclities' do
+          let(:user_request) do
+            iam_sign_in(FactoryBot.build(:iam_user, :no_vha_facilities))
+            VCR.use_cassette('payment_information/payment_information') do
+              VCR.use_cassette('user/get_facilities_no_ids', match_requests_on: %i[method uri]) do
+                get '/mobile/v0/user', headers: iam_headers
+              end
+            end
+          end
+
+          it 'authorized services does not include scheduleAppointments' do
+            user_request
+            expect(attributes['authorizedServices']).not_to include('scheduleAppointments')
+          end
+
+          it 'increments statsd' do
+            expect do
+              user_request
+            end.to trigger_statsd_increment('mobile.schedule_appointment.policy.failure', times: 1)
+          end
+        end
+
+        context 'due to not being LOA3' do
+          let(:user_request) do
+            iam_sign_in(FactoryBot.build(:iam_user, :loa2))
+            VCR.use_cassette('payment_information/payment_information') do
+              VCR.use_cassette('user/get_facilities_no_ids', match_requests_on: %i[method uri]) do
```

Comment:
Doesn't this mean both conditions are being met? If possible, I'd change this to the normal "get facilities" to ensure that it's failing for the right reason.

Reply:
Looks like loa2 changes the facility behavior. no facility ids are given in the call when loa2.